apps/s_time.c

```diff
@@ -60,12 +61,15 @@ const OPTIONS s_time_options[] = {
     {"cert", OPT_CERT, '<', "Cert file to use, PEM format assumed"},
     {"nameopt", OPT_NAMEOPT, 's', "Various certificate name options"},
     {"key", OPT_KEY, '<', "File with key, PEM; default is -cert file"},
+    {"CAfile", OPT_CAFILE, '<', "PEM format file of CA's"},
     {"CApath", OPT_CAPATH, '/', "PEM format directory of CA's"},
-    {"cafile", OPT_CAFILE, '<', "PEM format file of CA's"},
```

Comment:
This seems like a breaking change

Reply:
What was there is actually a bug, or at least an inconsistency.  It should have been `-CAfile` to be consistent with everywhere else, and even the (1.1.1) manual is inconsistent, calling it `-cafile` in the synopsis and `-CAfile` in the description.

I think that the best we can do to avoid breaking too much is to have both variants in there.  My absolute preference would be if it was possible to have "invisible" options (i.e. options that don't show with `-help`), to be used for bug compatibility of this sort.

Reply:
I just decided that this is a bug that doesn't need to be fixed *here*, so undoing that name option name fix